interface/super/edit_globals.php

```diff
@@ -373,8 +373,13 @@ function checkBackgroundServices()
     'help_file_name' => ""
 );
 $oemr_ui = new OemrUI($arrOeUiSettings);
+$serverConfig = new \OpenEMR\FHIR\Config\ServerConfig();
```

Comment:
would place this in use clause at top script

Reply:
Fixed